src/Factory/Api/Mastodon/Relationship.php

```diff
@@ -38,7 +38,8 @@ public function createFromContactId(int $contactId, int $uid): RelationshipEntit
 			$pcid,
 			$contact,
 			Contact\User::isBlocked($cid, $uid),
-			Contact\User::isIgnored($cid, $uid)
+			Contact\User::isIgnored($cid, $uid),
+			Contact\User::isIsBlocked($cid, $uid),
```

Comment:
What is this function name?

Reply:
That function tells you that the user is blocked by that given contact.